.github/workflows/rust-ci.yml

```
@@ -11,44 +10,44 @@ jobs:
-      run: "yarn install"
-
-    - name: "Compile the contracts and generate the TypeChain bindings"
-      run: "yarn typechain"
-
-      # Now we can check rust formatting and run tests
-    - name: Checking code format
-      run: cd ./packages/ciphernode && cargo fmt -- --check
-
-    - name: Run tests
-      run: "yarn ciphernode:test"
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)